envs/example/defaults-2.0.yml

```diff
@@ -228,6 +228,7 @@ neutron:
   network_vlan_ranges: ''
   tenant_network_type: vxlan
   arp_responder: false
+  l2_population: false
```

Comment:
l2pop=false and arp_responder=false are the default settings. Sometimes we might forget what the default settings are. Since we included arp_responder, we should also include l2_pop.